README.md

```diff
@@ -60,100 +60,6 @@ First ensure that the [Operator Lifecycle Manager](https://github.com/operator-f
 $ kubectl apply -f deploy/upstream
 ```
 
-#### Installing an operator using Marketplace
```

Comment:
Please don't remove this from here. Troubleshooting is basically "what do I do if things go wrong". If needed link to this section from the troubleshooting guide.

Reply:
@aravindhp restored Readme to it's original state. Used links from Readme in the troubleshooting guide.

Reply:
@anik120 I think we could still point to a troubleshooting guide in our Readme, we just don't want to remove things from the Readme when they were already useful.